### PR TITLE
Use new URL for MathJax

### LINKED ISSUE #2633: CGAL Doxygen doc: MathJax CDN shutting down
https://www.mathjax.org/cdn-shutting-down/

> The MathJax CDN hosted at cdn.mathjax.org will be shutting down on April 30, 2017. 
Current and future releases available on other CDN providers.

That did not break about usage because:

> [Updated 2017/04/10]
>
> We will implement a temporary redirect on cdn.mathjax.org to redirect users to cdnjs.

We will have to do something in our Doxygen configuration, because the 
temporary redirection from cdn.mathjax.com to another provider will 
disappear.

As far as I remember, the documentation at https://doc.cgal.org/ will not break, because we ship a copy of MathJax. But all locally compiled/downloaded documentation will have the issue.

### PR DESCRIPTION
**Pull request by sloriot**
## Release Management

* Affected package(s): doc
* Issue(s) solved (if any): fix #2633

### CODE REVIEW

File: Documentation/doc/CMakeLists.txt [hunk 1]
```diff
@@ -185,7 +185,7 @@ function(configure_doxygen_package CGAL_PACKAGE_NAME)
 endfunction()
 
 # set up the directories and variables
-set(CGAL_DOC_MATHJAX_LOCATION "https://cdn.mathjax.org/mathjax/latest"
+set(CGAL_DOC_MATHJAX_LOCATION "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2"
```
Comment (lrineau) [inline Documentation/doc/CMakeLists.txt]: Do we really want to set that variable? Recent versions of Doxygen have a default for `MATHJAX_RELPATH` that is already right. And we can trust the Doxygen author better than us for upgrading that version when new versions of MathJax are published.

Comment (sloriot) [reply Documentation/doc/CMakeLists.txt]: AFAIS, this is only use in resource files for doxygen 1.8.4, so this should be fine.

Comment (lrineau) [reply Documentation/doc/CMakeLists.txt]: If I grep the variable name, I see:
```
doc/resources/1.8.13/BaseDoxyfile.in
1574:MATHJAX_RELPATH        = ${CGAL_DOC_MATHJAX_LOCATION}
```
Is not that Doxygen configuration variable used?

Comment (sloriot) [reply Documentation/doc/CMakeLists.txt]: You're right, I fixed that in #2760 in 48658fe5f6 (since new resources files were added for 1.8.14).

**Comment by lrineau** [comment]
I do not know how to merge this PR in integration. There is a big conflict with `master` in `Documentation/doc/scripts/test_doxygen_versions.sh`.

**Comment by lrineau** [comment]
I have add a look at the result at https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-4.12-Ic-159/index.html and I have the impression that the CDN of MathJax is never used.

**Comment by sloriot** [comment]
In the testsuite we are using a local version of MathJax and not the online one.

**Comment by lrineau** [comment]
Alright. Now I understand: the cdn version of MathJax will only be used in:
  - local Doxygen builds by users or developers,
  - in the doc_html tarballs...
but that is not what we test automatically.

**Comment by lrineau** [comment]
@sloriot Can you please merge this PR in `master` and in the release branch?

**Comment by sloriot** [comment]
@lrineau: done.

**Pull request merged**